draft-kiefer-tls-ecdhe-sidh.md

```diff
@@ -192,6 +192,14 @@ key to octets.
 
 The SIDH shared secret is calculated as described in Section 1.3.6 of {{SIKE}}
 using `isoex_l`.
+
+Calculating SIDH shared secret requires each side to use isogenies of different
+degree. This document assumes parameterizations as described in {{SIKE}}, which
+is based on 4- and 3-power degree isogenies.
+In order to calculate shared secret, client always generates ephemeral keypair 
```

Comment:
"In order to calculate the shared secret, the client always generates an ephemeral keypair"...